joblib/test/test_memmapping.py

```diff
@@ -509,3 +510,19 @@ def test_pool_get_temp_dir(tmpdir):
     if sys.platform.startswith('win'):
         assert shared_mem is False
     assert pool_folder.endswith(pool_folder_name)
+
+
+@with_numpy
+@skipif(sys.platform == 'win32', reason='This test is not stable under windows'
```

Comment:
Update the reason here: this is not a problem of stability but more about PermissionError on Windows.